src/views/List.jsx

```diff
@@ -22,36 +36,57 @@ export function List({ data }) {
 		setFilterInput('');
 	};
 
-	return (
-		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-
-			<form>
-				<label htmlFor="list-filter">Filter items</label>
-				<br />
-				<div>
-					<input
-						id="list-filter"
-						type="text"
-						placeholder="Start typing here..."
-						value={filterInput}
-						onChange={handleInput}
-					/>
-					{filterInput && <button onClick={handleClick}>X</button>}
-				</div>
-			</form>
-			{/* Uses data or state of filteredList depending on state of filterInput */}
-			<ul>
-				{!filterInput
-					? data.map((item) => {
-							return <ListItem key={item.id} name={item.name} />;
-					  })
-					: filteredList.map((item) => {
-							return <ListItem key={item.id} name={item.name} />;
-					  })}
-			</ul>
-		</>
-	);
+	/*Handles navigation to Add Item view */
+	const handleAddItem = () => {
+		navigate('/add-item');
+	};
+
+	if (loading) {
```

Comment:
We can create a new issue here to look into adding loading spinner or something similar. Not needed right now.